.github/workflows/python.yaml

```diff
@@ -1,20 +1,20 @@
-# .github/workflows/tests.yml
-name: Tests
-on: pull_request
-jobs:
-  tests:
-    strategy:
-      matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9']
-        os: [ubuntu-latest, windows-latest, macos-latest]
-    runs-on: ${{ matrix.os }}
-    name: Python ${{ matrix.python-version }}
-    steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-python@v1
-      with:
-        python-version: ${{ matrix.python-version }}
-        architecture: x64
-    - run: pip install nox==2019.11.9
-    - run: pip install poetry==1.0.5
-    - run: nox
+# .github/workflows/tests.yml
+name: Tests
+on: pull_request
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.6', '3.7', '3.8', '3.9']
```

Comment:
Remove 3.6 and add 3.10 here please. Windows runner no longer supports 3.6 and it has been EOL for some time now.